scripts/python/common_setup.py

```diff
@@ -251,33 +251,67 @@ def mkldnn_build_dir(root=None):
 def mkl_install_dir(root):
     return os.path.join(mkldnn_build_dir(root), "intel")
 
+def acl_root_dir(root):
+    return os.path.join(mkldnn_build_dir(root), 'acl', 'ComputeLibrary')
 
-def config_mkldnn(root, cxx11_abi):
+def config_mkldnn(root, args):
     build_dir = mkldnn_build_dir(root)
     ensure_empty_dir(build_dir, clear_hidden=False)
     mkl_dir = mkl_install_dir(root)
+    acl_dir = acl_root_dir(root)
     ensure_empty_dir(mkl_dir, clear_hidden=False)
-    # download mkl-lib/include
-    with cwd(mkl_dir):
-        download_cmd = """
-          unset HTTPS_PROXY
-          curl -fsSL https://hlomodule.oss-cn-zhangjiakou.aliyuncs.com/mkl_package/mkl-static-2022.0.1-intel_117.tar.bz2  | tar xjv
-          curl -fsSL https://hlomodule.oss-cn-zhangjiakou.aliyuncs.com/mkl_package/mkl-include-2022.0.1-h8d4b97c_803.tar.bz2 | tar xjv
-        """
-        execute(download_cmd)
+    ensure_empty_dir(acl_dir, clear_hidden=False)
+    if args.x86:
+        with cwd(mkl_dir):
+            # download mkl-lib/include
+            download_cmd = """
+              unset HTTPS_PROXY
+              curl -fsSL https://hlomodule.oss-cn-zhangjiakou.aliyuncs.com/mkl_package/mkl-static-2022.0.1-intel_117.tar.bz2  | tar xjv
+              curl -fsSL https://hlomodule.oss-cn-zhangjiakou.aliyuncs.com/mkl_package/mkl-include-2022.0.1-h8d4b97c_803.tar.bz2 | tar xjv
+            """
+            execute(download_cmd)
+
+    if args.aarch64:
+        with cwd(acl_dir):
+            # downlaod and build acl for onednn
+            cmd = '''
+              readonly ACL_REPO="https://github.com/ARM-software/ComputeLibrary.git"
```

Comment:
Shall we just built ACL into docker image? So does to MKL ...

Reply:
we can not pre-build this because currently oneDNN  repo relies on building ACL from source. And we do not want to use a customized oneDNN version.